### PR TITLE
Modified lambda_for_fileDelete

### LINKED ISSUE #38: Deleting file should also delete corresponding url
**Describe the bug**
While running the deleteFile lambda, the corresponding URL for the file doesn't get deleted.

Area Where Bug Occurred
- [x] lambda_for_fileDelete

**Before submitting an issue check the below**
- [x] I have synced all my node versions as mentioned in the project"
- [x] I am using the same version of npm as is the project

### PR DESCRIPTION
**Pull request by lakshya-20**
Modified `lambda_for_fileDelete`. Now, all the existing URLs corresponding to the file which is to be deleted will also be deleted.

### CODE REVIEW

File: lambdas/file_access_patterns/lambda_for_fileDelete/index.js [hunk 1]
```diff
@@ -43,12 +53,52 @@ exports.deleteFile = async (event) => {
...
+      return response(201,"File delete success")
+    }
   }
   catch(err){
     return response(err.statusCode,err.message)
   }
+
+  url_data.forEach(request_item_arr_util)
+  params={
+    "RequestItems":{
```
Comment (danishjamal104) [inline lambdas/file_access_patterns/lambda_for_fileDelete/index.js:89]: can we pass a direct object to `RequestItems` key, or we have to pass an `array`?

**Review by danishjamal104 — COMMENTED** [review]
Rest everything is fine

**Comment by danishjamal104** [comment]
please update the title of PR, @lakshya-20 . Thanks :)

File: lambdas/file_access_patterns/lambda_for_fileDelete/index.js (hunk 1 above)
Comment (lakshya-20) [reply lambdas/file_access_patterns/lambda_for_fileDelete/index.js:89]: We have to pass an array because there can be multiple items to delete.

Comment (danishjamal104) [reply lambdas/file_access_patterns/lambda_for_fileDelete/index.js:89]: so it will work fine!!

Comment (danishjamal104) [reply lambdas/file_access_patterns/lambda_for_fileDelete/index.js:89]: so it will work fine!!

**Review by danishjamal104 — APPROVED** [review]

**Review by ayush0x00 — APPROVED** [review]

**Pull request merged**